src/Stratis.Bitcoin.Features.BlockStore/BlockStoreQueue.cs

```diff
@@ -315,6 +322,7 @@ public void AddToPending(ChainedHeaderBlock chainedHeaderBlock)
             }
 
             this.blocksQueue.Enqueue(chainedHeaderBlock);
+            this.blocksQueueSizeBytes += chainedHeaderBlock.Block.BlockSize.Value;
```

Comment:
Not sure if we need `blocksQueueSizeBytes `

It will always be 0 except for time when batch is being saved. 

It would make more sense to measure how much time it took to persist the batch.

Reply:
Fair point